src/test/c/functions_spec.c

```diff
@@ -88,13 +93,75 @@ void test_sum_vectors_multiple_elements() {
+  assert(tangent_forces[1], 4.275960624d, "test_compute_forces_one_element - resultant_forces[P2].y_component");
+  #undef size
+  #undef contacts_size
+}
+void
+test_compute_forces_elements() {
+  #define size 9
+  #define contacts_size 3
+  ParticleProperties properties[size] = { { 0.049, 247435.829652697, 19033.5253578998 }, { 0.049, 247435.829652697, 19033.5253578998 },{ 0.049, 247435.829652697, 19033.5253578998 },{ 0.049, 247435.829652697, 19033.5253578998 },{ 0.049, 247435.829652697, 19033.5253578998 },{ 0.049, 247435.829652697, 19033.5253578998 }, { 0.049, 247435.829652697, 19033.5253578998 },{ 0.049, 247435.829652697, 19033.5253578998 },{ 0.049, 247435.829652697, 19033.5253578998 },};
+  Particle particles[size] = { { 24.9996682317,	25, 50 }, { 24.3329247490,	74.1788246441, 50 },{ 20.8181703235, 122.9449251651, 50 },{ 16.8606509998, 172.4918861911, 50 },{ 75.0003317683, 25, 50 },{ 75.6670752510, 74.1788246441, 50 },{ 79.1818296765, 122.9449251651, 50 },{ 83.1393490002, 172.4918861911, 50 },{ 50, 75.7713467697, 50 } };
+  Vector velocities[size] = { { -0.00728767793878, 0 }, {-10.44576385514790, -9.61529833378254 },{-22.72352179173930, -6.11340796404052 },{ -22.22735951506210, -4.22438903830371 },{ 0.00728767793878, 0 },{ 10.44576385514790, -9.61529833378254 },{ 22.72352179173930, -6.11340796404052 },{ 22.22735951506210, -4.22438903830371 },{ -0.00000000000001, -259.77483981875700 }};
+  double normal_forces[size * size] = { 0, 143.165149, 0, 0, 0, 0, 0, 0,0, 0, 0, 0, 0, 0, 0, 0, 0, 6205.981479,0, 297.810731, 0, 81.040751, 0, 0, 0, 0, 0,0, 0, 81.040751, 0, 0, 0, 0, 0, 0,0, 0, 0, 0, 0, 143.165149, 0, 0, 0,0, 0, 0, 0, 143.165149, 0, 297.810731, 0, 6205.981479,0, 0, 0, 0, 0, 297.810731, 0, 81.040751, 0,0, 0, 0, 0, 0, 0, 81.040751, 0, 0,0, 6205.981479, 0, 0, 0, 6205.981479, 0, 0, 0 };
+  double tangent_forces[size * size] = { 0, 7.77476, 0, 0, 0, 0, 0, 0, 0, 1, 0, 1, 0, 0, 0, 0, 0, -553.79557,0, 61.408871, 0, 46.788899, 0, 0, 0, 0,0,0, 0, 46.788899128, 0, 0, 0, 0, 0, 0,0, 0, 0, 0, 0, -7.7747607, 0, 0, 0,0, 0, 0, 0, -7.77476, 0, -61.4089, 0, 553.7956,0, 0, 0, 0, 0, -61.408871, 0, -46.788899, 0,0, 0, 0, 0, 0, 0, -46.78890, 0, 0,0, -553.7956, 0, 0, 0, 553.7956, 0, 0, 0};
+
+  double dt = 0.000025;
+  Contact contacts[contacts_size] = { { 1, 0, 49.18334413},{ 1, 2, 48.89259718},{ 1, 8, 25.71643207}};
+  Vector resultant_forces[size] = { { 0 } };
+
+  compute_forces(dt, size, contacts_size, particles, properties, contacts, velocities, normal_forces, tangent_forces, resultant_forces);
+
+  // P1.
+  assert(resultant_forces[0].x_component, -6221.088058d, "test_compute_forces_elements- resultant_forces[P1].x_component");
+  assert(resultant_forces[0].y_component, -1131.3036302d, "test_compute_forces_elements - resultant_forces[P1].y_component");
+  assert(normal_forces[1], 201.7636d, "test_compute_forces_elements - resultant_forces[P1].y_component");
```

Comment:
`test_compute_forces_elements` should be `test_compute_forces_multiple_elements` as in other functions tests.